### PR TITLE
New pages: HTMLSourceElement api pages

### PR DESCRIPTION
**Pull request by estelle**
part of https://github.com/openwebdocs/project/issues/214

### CODE REVIEW

File: files/en-us/web/api/htmlsourceelement/srcset/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+title: "HTMLSourceElement: srcset property"
+short-title: srcset
+slug: Web/API/HTMLSourceElement/srcset
+page-type: web-api-instance-property
+browser-compat: api.HTMLSourceElement.srcset
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`srcset`** property of the {{domxref("HTMLSourceElement")}} interface is a string containing a comma-separated list of candidate images and descriptors. Each list item in the comma-separated list includes the URL of an image resource to use as the source for the element and the circumstances under which that image should be used. Each image in the comma selist one or more image URLs and their descriptorseach specifying image resources to use under given circumstances. Each candidate image is a URL followed by the width of the image with a `'w'` or the pixel density with an `'x'`.
```
Comment (wbamberg) [inline files/en-us/web/api/htmlsourceelement/srcset/index.md:11]:
```suggestion
The **`srcset`** property of the {{domxref("HTMLSourceElement")}} interface is a string containing a comma-separated list of candidate images.

Each candidate image includes the URL of an image resource to use as the source for the element and optionally a descriptor indicating the circumstances in which the image should be used. The descriptor is either a number followed by `'w'`, indicating the element width, or a number followed by `'x'`, indicating the device pixel density.
```

File: files/en-us/web/api/htmlsourceelement/srcset/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+It reflects the `srcset` attribute of the {{HTMLElement("source")}} element nested in a {{htmlelement("picture")}} element. It has no meaning and is ignored when it is nested in an {{htmlelement("audio")}} or {{htmlelement("video")}} element, which use the {{domxref("HTMLSourceElement.src", "src")}} instead.
```
Comment (wbamberg) [inline files/en-us/web/api/htmlsourceelement/srcset/index.md:13]:
```suggestion
It reflects the `srcset` attribute of the {{HTMLElement("source")}} element nested in a {{htmlelement("picture")}} element. It has no meaning and is ignored when it is nested in an {{htmlelement("audio")}} or {{htmlelement("video")}} element, which use the {{domxref("HTMLSourceElement.src", "src")}} attribute instead.
```

File: files/en-us/web/api/htmlsourceelement/type/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+title: "HTMLSourceElement: type property"
+short-title: type
+slug: Web/API/HTMLSourceElement/type
+page-type: web-api-instance-property
+browser-compat: api.HTMLSourceElement.type
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`type`** property of the {{domxref("HTMLSourceElement")}} interface is a string representing the type of media of the resource.
```
Comment (wbamberg) [inline files/en-us/web/api/htmlsourceelement/type/index.md:11]:
```suggestion
The **`type`** property of the {{domxref("HTMLSourceElement")}} interface is a string representing the {{glossary("MIME type")}} of the media resource.
```

File: files/en-us/web/api/htmlsourceelement/type/index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  id="el"
+  src="large.webp"
+  type="video/webp"
+  media="screen and (min-width: 600px)" />
+```
+
+```js
+const el = document.getElementById("el");
+console.log(el.type); // Output: "video/webp"
+el.type = "image/webp"; // Updates the type value
```
Comment (wbamberg) [inline files/en-us/web/api/htmlsourceelement/type/index.md:32]: This (updating the type for an element) seems like a strange thing to do? Maybe more likely would be to create a new source element from scratch?

File: files/en-us/web/api/htmlsourceelement/media/index.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+title: "HTMLSourceElement: media property"
+short-title: media
+slug: Web/API/HTMLSourceElement/media
+page-type: web-api-instance-property
+browser-compat: api.HTMLSourceElement.media
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`media`** property of the {{domxref("HTMLSourceElement")}} interface is a string representing a list of one or more media formats to which the resource applies.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/api/htmlsourceelement/media/index.md]: i copied it from htmllinkelement, which should be updated (not in this PR). changed to:

> The **`media`** property of the {{domxref("HTMLSourceElement")}} interface is a string representing the intended destination medium for the resource. The value is a [media query](/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries), which is a comma separated list of media-types, media-features, and logical operators.

File: files/en-us/web/api/htmlsourceelement/type/index.md (hunk 4 above)
Comment (estelle) [reply files/en-us/web/api/htmlsourceelement/type/index.md]: removed it.

**Review by estelle — COMMENTED** [review]
THanks @wbamberg

**Review by wbamberg — APPROVED** [review]
👍 looks great.

**Pull request merged**